Makefile

```diff
@@ -5,6 +5,8 @@ WARN_COLOR=\033[33;01m
 
 VERSION ?= "0.0.0-dev-$(shell git rev-parse --short HEAD)"
 
+GOOGLEAPIS_DIR ?= "../googleapis"
```

Comment:
Maybe add a comment so we know where to get the content for this directory?

Reply:
agree, added